Making Core versions page shorter and more user centric (#8663)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/core-versions.md

- To learn about installing <Constant name="core" />, see "[How to install <Constant name="core" />](/docs/core/installation-overview)."
- To restrict your project to only work with a range of dbt Core versions, or use the currently running dbt Core version, see [`require-dbt-version`](/reference/project-configs/require-dbt-version) and [`dbt_version`](/reference/dbt-jinja-functions/dbt_version).

## Version support prior to v1.0

All <Constant name="core" /> versions released prior to 1.0 and their version-specific documentation have been deprecated. If upgrading to a currently supported version, reference our [best practices for upgrading](#best-practices-for-upgrading)
- [Choosing a <Constant name="core" /> version in <Constant name="cloud" />](/docs/dbt-versions/upgrade-dbt-version-in-cloud)
- [How to install <Constant name="core" />](/docs/core/installation-overview)
- [`require-dbt-version`](/reference/project-configs/require-dbt-version) and [`dbt_version`](/reference/dbt-jinja-functions/dbt_version)

> ## EOL version support 

Author: need to add redirects

pagination_prev: null
---

> If you're using <Constant name="dbt_platform" />, you don't need to manage dbt versions yourself. [Release tracks](/docs/dbt-versions/cloud-release-tracks) keep you up to date automatically, with early access to new features before they're available in <Constant name="core" />.

Reviewer: great clarification!

> If you're using <Constant name="dbt_platform" />, you don't need to manage dbt versions yourself. [Release tracks](/docs/dbt-versions/cloud-release-tracks) keep you up to date automatically, with early access to new features before they're available in <Constant name="core" />.

Reviewer: i feel like we should lead with what this page is and who it's for rather than who it's not for. e.g.
```suggestion
This page describes versioning for the <Constant name="core"/> engine (Python-based CLI). If you run the <Constant name="core"/> engine locally (for example, using `pip`), then this page is for you. <Constant name="core"/> releases follow [semantic versioning](https://semver.org/).

If you're using <Constant name="dbt_platform" /> (including the <Constant name="cloud_cli"/>, you don't need to manage dbt versions yourself. [Release tracks](/docs/dbt-versions/cloud-release-tracks) automatically keep you up to date and provide early access to new features before they’re available in <Constant name="core" />.
```

If you manage your own <Constant name="core" /> versions locally, read on. <Constant name="core" /> releases follow [semantic versioning](https://semver.org/).

<Snippet path="core-version-support" />

<Snippet path="core-versions-table" />

### Further reading

- [Choosing a <Constant name="core" /> version in <Constant name="cloud" />](/docs/dbt-versions/upgrade-dbt-version-in-cloud)
- [How to install <Constant name="core" />](/docs/local/install-dbt)

> - [`require-dbt-version`](/reference/project-configs/require-dbt-version) and [`dbt_version`](/reference/dbt-jinja-functions/dbt_version)

Reviewer: do we feel the links to require dbt version and dbt version are enough info for the user? i feel like the old info was useful so the readers has a heads up on what they are?

Author: I will add some more context! thanks Mirna

Author: Context added [here](https://github.com/dbt-labs/docs.getdbt.com/pull/8663/commits/8aaae80c9158b8c96198a8ecad69b73dd1b7a316)

## End-of-life versions

Once a <Constant name="core" /> version reaches end-of-life (EOL), it no longer receives patches, including for known bugs. We recommend upgrading to a newer version in [<Constant name="cloud" />](/docs/dbt-versions/upgrade-dbt-version-in-cloud) or [<Constant name="core" />](/docs/local/install-dbt#upgrading-dbt-core). All versions prior to v1.0 have been deprecated.

## Current version support

> dbt supports each minor version (for example, v1.8) for **one year** from its initial release. During that window, we release patches with bug fixes and security updates. When we refer to a minor version, we mean its latest available patch (v1.8.x).

Reviewer: italics for emphasis per style guide
```suggestion
dbt supports each minor version (for example, v1.8) for _one year_ from its initial release. During that window, we release patches with bug fixes and security updates. When we refer to a minor version, we mean its latest available patch (v1.8.x).
```

pagination_prev: null
---

> This page describes versioning for the <Constant name="core"/> engine (Python-based CLI). If you run the <Constant name="core"/> engine locally (for example, using `pip`), then this page is for you. <Constant name="core"/> releases follow [semantic versioning](https://semver.org/).

Author: ```suggestion
Learn about versioning for the <Constant name="core"/> engine (Python-based CLI). If you run the <Constant name="core"/> engine locally (for example, using `pip`), then this page is for you. <Constant name="core"/> releases follow [semantic versioning](https://semver.org/).
:::note
If you're interested in using <Constant name="fusion_engine"/>, locally or in <Constant name="dbt_platform"/> then read [Get Started](/docs/local/install-dbt?version=2.0#get-started).
:::
```

:::note
If you're interested in using <Constant name="fusion_engine"/>, locally or in <Constant name="dbt_platform"/>, then read [Get Started](/docs/local/install-dbt?version=2.0).
:::

If you're using <Constant name="dbt_platform" /> (including the <Constant name="cloud_cli"/>, you don't need to manage dbt versions yourself. [Release tracks](/docs/dbt-versions/cloud-release-tracks) automatically keep you up to date and provide early access to new features before they’re available in <Constant name="core" />.

If you manage your own <Constant name="core" /> versions locally, read on. <Constant name="core" /> releases follow [semantic versioning](https://semver.org/).

<Snippet path="core-version-support" />

<Snippet path="core-versions-table" />

### Further reading

- [Choosing a <Constant name="core" /> version in <Constant name="cloud" />](/docs/dbt-versions/upgrade-dbt-version-in-cloud): Learn how you can use <Constant name="core" /> versions in <Constant name="cloud" />.

> - [How to install <Constant name="core" />](/docs/local/install-dbt): Learn about installing <Constant name="core" />

Author: ```suggestion
- [How to install <Constant name="core" />](/docs/local/install-dbt): Learn about installing <Constant name="core" />.
```

pagination_prev: null
---

Learn about versioning for the <Constant name="core"/> engine (Python-based CLI). If you run the <Constant name="core"/> engine locally (for example, using `pip`), then this page is for you. <Constant name="core"/> releases follow [semantic versioning](https://semver.org/).

If you're using <Constant name="dbt_platform" /> (including the <Constant name="cloud_cli"/>, you don't need to manage dbt versions yourself. [Release tracks](/docs/dbt-versions/cloud-release-tracks) automatically keep you up to date and provide early access to new features before they’re available in <Constant name="core" />.

> :::note

Author: @mirnawong1 I added this note. WDYT?

  - snowflake: 1.9.0 - Up to date!
```

You can also find the registered adapter version in [logs](/reference/global-configs/logs). For example, in `logs/dbt.log`:

```
[0m13:13:48.572182 [info ] [MainThread]: Registered adapter: snowflake=1.9.0
```

> Refer to [Supported Data Platforms](/docs/supported-data-platforms) for the full list of adapters.

Reviewer: ```suggestion
Refer to [Supported data platforms](/docs/supported-data-platforms) for the full list of adapters.
```
